src/FSharpLint.Core/Rules/Conventions/Naming/UnnestedFunctionNames.fs

```diff
@@ -0,0 +1,36 @@
+module FSharpLint.Rules.UnnestedFunctionNames
+
+open FSharp.Compiler.Syntax
+open FSharpLint.Framework.Ast
+open FSharpLint.Framework.AstInfo
+open FSharpLint.Framework.Rules
+open FSharpLint.Rules.Helper.Naming
+
+let private getFunctionIdents _ =
+    function
+    | SynPat.LongIdent (longIdent, _, _, pats, _, _) ->
+        match pats with
+        | SynArgPats.Pats _ ->
+            match List.tryLast longIdent.Lid with
+            | Some ident -> (ident, ident.idText, None) |> Array.singleton
+            | None -> Array.empty
+        | _ -> Array.empty
+    | _ -> Array.empty
```

Comment:
this function is exactly the same in UnnestedFunctionNames.fs and NestedFunctionNames.fs, let's place it in a common place to share it between the two

Reply:
I've placed the function in a common place and updated the PR. Could you please take a look? thanks